pom.xml

```diff
@@ -228,7 +252,7 @@
         <artifactId>findbugs-maven-plugin</artifactId>
         <configuration>
           <skip>${skipFindbugs}</skip>
```

Comment:
Can be also replaced by the Parent POM implementation

Reply:
done